src/Administration/Resources/app/administration/src/app/component/entity/sw-one-to-many-grid/index.js

```diff
@@ -182,7 +198,9 @@ Component.extend('sw-one-to-many-grid', 'sw-data-grid', {
             return this.repository
                 .syncDeleted(selectedIds, this.result.context)
                 .then(() => {
-                    return this.deleteItemsFinish();
+                    this.resetSelection();
```

Comment:
is this intended that this.$emit('items-delete-finish'); is no longer emitter?

Reply:
Yes, if this.$emit('items-delete-finish') calls before the load method, it will lead to the error that the list is not loading when we delete all items of a lasted page. A solution like the current `deleteItem` method.